services/registration/controller/controller.go

```diff
@@ -207,20 +207,20 @@ func UpdateCurrentUserRegistration(w http.ResponseWriter, r *http.Request) {
 	err = service.UpdateUserRegistration(id, user_registration)
 
 	if err != nil {
-		panic(errors.UnprocessableError(err.Error()))
+		panic(errors.InternalError(err.Error(), "Could not update user's registration.\n"+err.Error()))
```

Comment:
Don't need a newline here and don't need to append `err.Error()` since it will be included in the `RawError` field.